implementation/service_discovery/src/service_discovery_impl.cpp

```diff
@@ -1543,7 +1545,7 @@ service_discovery_impl::process_offerservice_serviceentry(
                                     << std::setw(4) << _instance << "."
                                     << std::setw(4) << eg << "]"
                                     << " using reliability type:  "
-                                    << std::setw(4) << (uint16_t) offer_type;
+                                    << std::setw(4) << +static_cast<uint16_t>(offer_type);
```

Comment:
Remove typo.

```suggestion
                                    << std::setw(4) << static_cast<uint16_t>(offer_type);
```

Reply:
Not a typo, see https://github.com/COVESA/vsomeip/pull/560#discussion_r1550589301